udata_front/theme/gouvfr/assets/js/components/MarkdownEditor/Milkdown/LinkPreview/useLinkPreview.ts

```diff
@@ -0,0 +1,20 @@
+import { inject, readonly } from "vue";
+import { type InjectionKey, type Ref, provide, ref } from "vue";
+
+export function useLinkPreviewProvider() {
```

Comment:
When adding a link in the editor body directly `[this is a link](https://example.com)`, we don't have a link rendering. It may be confusing for people who may write in Markdown since other stuffs get interpreted.

Reply:
I'll look into this.

Reply:
I don't think this is easily possible to do (without a custom plugin). The `[]()` are escaped to be shown in the WYSIWYG editor. We can improve the UX in an upcoming PR with a floating menu with common actions on selected content like bold, italic and link.